elements/howto-tabs/howto-tabs.e2etest.js

```diff
@@ -8,103 +10,84 @@ describe('howto-tabs', function() {
       .then(_ => helper.waitForElement(this.driver, 'howto-tabs'));
   });
 
-  it('should focus the next tab on [arrow right]',
-    async function() {
-      const found =
-        await helper.pressKeyUntil(
-          this.driver,
-          Key.TAB,
-          `return document.activeElement.getAttribute('role') === 'tab';`
-        );
-      expect(found).to.equal(true);
+  it('should focus the next tab on [arrow right]', async function() {
```

Comment:
does this test assert that the next tab was actually focused? Couldn't it in theory be any tab other than the current one?